scripts/beachball/tagVNext.ts

```diff
@@ -0,0 +1,70 @@
+import { execSync } from 'child_process';
+import * as semver from 'semver';
+import * as tmp from 'tmp';
+import * as fs from 'fs';
+import { AllPackageInfo, getAllPackageInfo, isConvergedPackage } from '../monorepo/index';
+
+function tagPackages() {
+  const packagesToTag = getPackagesToTag();
+  const { path: npmrcPath, cleanup: cleanupTmpFiles } = createTmpNpmrc();
+
+  packagesToTag.forEach(pkg => {
+    tagPackage(pkg.name, pkg.version, npmrcPath);
+  });
+
+  cleanupTmpFiles();
```

Comment:
There are options you can use to clean up automatically. A global function call and an option when creating the dir (look at projects-test utils).